packages/loader/container-loader/src/container.ts

```diff
@@ -584,14 +588,36 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             loader.services.subLogger,
             undefined,
             {
-                all:{
+                all: {
                     clientType, // Differentiating summarizer container from main container
                     loaderVersion: pkgVersion,
                     containerId: uuid(),
                     docId: () => this.id,
                     containerAttachState: () => this._attachState,
                     containerLoaded: () => this.loaded,
                 },
+                error: {
+                    initialSequenceNumber: () => this._deltaManager?.initialSequenceNumber,
+                    lastKnownSeqNumber: () => this._deltaManager?.lastKnownSeqNumber,
+                    lastSequenceNumber: () => this._deltaManager?.lastSequenceNumber,
```

Comment:
lastQueuedSequenceNumber is a good one to log too. It is private in DeltaManager though.

Reply:
i've reduce the scope here a bit, so i think that's a bit out of scope know